.drone-v1.yml

```diff
@@ -0,0 +1,268 @@
+---
+kind: pipeline
+type: kubernetes
+name: build
+
+steps:
+  - name: install dependancies
+    image: node:14.15.4-alpine
+    commands:
+      - npm --loglevel warn install --production=false --no-optional
+
+  - name: build project
+    image: node:14.15.4-alpine
+    commands:
+      - npm run build-prod
+    depends_on:
+      - install dependancies
+
+  - name: test project
+    image: node:14.15.4-alpine
+    commands:
+      - npm test
+    depends_on:
+      - install dependancies
+
+  - name: lint project
+    image: node:14.15.4-alpine
+    commands:
+      - npm run lint
+    depends_on:
+      - install dependancies
+
+  - name: sonar scanner
+    image: quay.io/ukhomeofficedigital/sonar-scanner:v3.0.2
+    depends_on:
+      - build project
+      - test project
+      - lint project
```

Comment:
```suggestion
```
I think we want failing lints to go to Sonarqube, if only so Sonarqube can also tell the developer to go fix the linting issues.

Reply:
Done - although sonarcube is just broken at present (will create a ticket to look at) 👍🏼